release-notes/1.20.1.md

```diff
@@ -0,0 +1,27 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.20.1.
+
+Please see [component versions](https://anbox-cloud.io/docs/ref/component-versions) for a list of updated components.
+
+## New features & improvements
+
+* A new `amc image sync` command is available to explicitly trigger download of an image from the image server. <!--AC-2085-->
+* The Anbox Management Service (AMS) derives instance architecture from the architecture of the selected image instead of the Application Binary Interface (ABI).<!--AC-2138-->
+* Android security updates for December 2023 (see [Android Security Bulletin - December 2023](https://source.android.com/docs/security/bulletin/2023-12-01) for more information).<!--AC-2127-->
+* The Android WebView has been updated to [120.0.6099.43](https://chromereleases.googleblog.com/2023/11/early-stable-update-for-android.html).
+* `amc` returns better error details when the AMS client is not registered to the LXD cluster. <!--AC-2097-->
+* When an instance is being created, better status messages that give information on the steps of instance creation are displayed. These status messages are cleared once the instance moves to running status. <!--AC-2120-->
+* In the AMS HTTP API, the `ErrorMessage` field is moved from the `types.Image` to the `types.ImageVersion` object. <!--AC-2047-->
+
+## Bugs fixed
+* Post an Anbox Cloud appliance upgrade, streaming fails because the stream agent and gateway could not communicate with each other. <!--AC-2139-->
+* Instances are not able to start the nested Android LXD container.<!--AC-2136-->
+* Base instances are available for deletion on the *Instances* list page of the web dashboard. <!--AC-2146-->
+* The web dashboard allows to edit immutable applications, for example, an application synchronised from the application registry. <!--AC-2144-->
+* LXD charm fails in `api-relation-changed` hook. <!--AC-2070-->
+* A session that has an error status is stuck with a connecting indicator instead of displaying the error details. <!--AC-2141-->
+
+## Upgrade instructions
+
+See [Upgrade Anbox Cloud](https://anbox-cloud.io/docs/howto/update/upgrade-anbox) or [Upgrade the Anbox Cloud Appliance](https://anbox-cloud.io/docs/howto/update/upgrade-appliance) for instructions on how to update your Anbox Cloud deployment to the 1.20.1 release.
```

Comment:
````suggestion
See [Upgrade Anbox Cloud](https://anbox-cloud.io/docs/howto/update/upgrade-anbox) or [Upgrade the Anbox Cloud Appliance](https://anbox-cloud.io/docs/howto/update/upgrade-appliance) for instructions on how to update your Anbox Cloud deployment to the 1.20.1 release.

## Known issues

Since 1.20.0, Android 11 containers fail to start with pure software rendering (i.e., without any GPU attached) because of a missing library. This will be fixed in 1.20.2.

Affected users can solve the issue by installing the missing package in a pre-start [hook](https://anbox-cloud.io/docs/ref/hooks):
```
#!/bin/sh -ex
if  [ "$CONTAINER_TYPE" = "regular" ]; then
  exit 0
fi
apt update
apt install --no-install-recommends -y libgl1-mesa-dri
```
````

Reply:
LGTM

Reply:
LGTM. 
One nitpick. Let's use instances rather than container in the known issue, since the deb package would be missing in the VM LXD too.